src/test/unit_tests/jest_tests/test_nc_lifecycle_posix_integration.test.js

```diff
@@ -1164,6 +1170,7 @@ async function create_object(sdk, bucket, key, size, is_old, tagging) {
 
 /**
  * update_file_mtime updates the mtime of the target path
+ * Warning - this operation would change the mtime of the file to 5 days ago - which means that it changes the etag / obj_id of the object
```

Comment:
maybe also add a warning about it not changing the version_id so it will cause problems with versioning

Reply:
Thanks, I will add.